bson/src/test/unit/org/bson/types/ObjectIdTest.java

```diff
@@ -88,12 +89,12 @@ public void testDateCons() {
 
     @Test
     public void testMachineIdentifier() {
-        assertTrue(ObjectId.getGeneratedMachineIdentifier() > 0);
+        assertTrue(ObjectId.getGeneratedMachineIdentifier() == 0);
         assertEquals(0, ObjectId.getGeneratedMachineIdentifier() & 0xff000000);
 
         assertEquals(5, new ObjectId(0, 5, (short) 0, 0).getMachineIdentifier());
         assertEquals(0x00ffffff, new ObjectId(0, 0x00ffffff, (short) 0, 0).getMachineIdentifier());
-        assertEquals(ObjectId.getGeneratedMachineIdentifier(), new ObjectId().getMachineIdentifier());
+        // assertEquals(ObjectId.getGeneratedMachineIdentifier(), new ObjectId().getMachineIdentifier());
```

Comment:
Since I removed the generation of a machine ID, the assert is no longer valid.

Reply:
I think we'll be able to add this back

Reply:
I added back the commented assert.